admin_guide/cluster_capacity.adoc

```diff
@@ -0,0 +1,251 @@
+[[admin-guide-cluster-capacity]]
+= Analyzing Cluster Capacity
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+As a cluster administrator, you can use the cluster capacity tool to view the
+number of pods that can be scheduled in order to increase the current resources
+before they become exhausted, and to ensure any future pods can be scheduled.
+This capacity comes from an individual node host in a cluster, and includes CPU,
+memory, disk space, and others.
+
+The cluster capacity tool simulates a sequence of scheduling decisions to
+determine how many instances of an input pod can be scheduled on the cluster
+before it is exhausted of resources in order to provide a more accurate
```

Comment:
Same as previous `in order to` - Use "to" in most cases to avoid wordiness. (Style guide)